pkg/ddevapp/utils.go

```diff
@@ -419,3 +419,8 @@ func (app *DdevApp) GetRelativeWorkingDirectory() string {
 	pwd, _ := os.Getwd()
 	return app.GetRelativeDirectory(pwd)
 }
+
+func (app *DdevApp) IsCustomCert() bool {
+	customCertsPath := app.GetConfigPath("custom_certs")
+	return fileutil.FileExists(filepath.Join(customCertsPath, fmt.Sprintf("%s.crt", app.Name)))
+}
```

Comment:
Let's use the better name `HasCustomCert`, and extract the logic for http/https:

```suggestion
// HasCustomCert returns true if the project uses a custom certificate
func (app *DdevApp) HasCustomCert() bool {
	customCertsPath := app.GetConfigPath("custom_certs")
	return fileutil.FileExists(filepath.Join(customCertsPath, fmt.Sprintf("%s.crt", app.Name)))
}

// CanUseHTTPOnly returns true if the project can be accessed via http only
func (app *DdevApp) CanUseHTTPOnly() bool {
	return !nodeps.IsGitpod() &&
		!nodeps.IsCodespaces() &&
		(globalconfig.GetCAROOT() == "" || IsRouterDisabled(app)) &&
		!app.HasCustomCert()
}
```